src/main/java/igrad/model/module/ModuleCode.java

```diff
@@ -11,9 +11,9 @@
 
 
     public static final String MESSAGE_CONSTRAINTS =
-        "Module code should contain two letters at the front and four "
+        "Module code should contain at two or three letters at the front and four "
```

Comment:
"two or three" ?

Reply:
think he missed out 'least' ttoo

Reply:
"Module code should contain two or three letters at the front and four "

Reply:
oh ok sry!

Reply:
i think some  module codes have that